cuqi/experimental/mcmc/_pcn.py

```diff
@@ -88,4 +88,10 @@ def dim(self): # TODO. Check if we need this. Implemented in base class
         return self._dim
 
     def tune(self, skip_len, update_count):
-        pass
+
```

Comment:
Could you please add a bit of explanation here of how this tuning works/what each of the lines do?

Reply:
Added references and comments on the codes as suggested.